tests/puppeteer/campaigns/sanity/01_installShop/01_installShop.js

```diff
@@ -27,39 +29,45 @@ describe('Install Prestashop', async () => {
   // Steps
   it('should open the Install page', async function () {
     await this.pageObjects.installPage.goTo(global.INSTALL.URL);
-    await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.firstStepPageTitle,
+    const result = await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.firstStepPageTitle,
       [this.pageObjects.installPage.firstStepFrTitle, this.pageObjects.installPage.firstStepEnTitle]);
+    await expect(result).to.be.true;
   });
   it('should change language to English and check title', async function () {
     await this.pageObjects.installPage.setInstallLanguage();
-    await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.firstStepPageTitle,
+    const result = await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.firstStepPageTitle,
       this.pageObjects.installPage.firstStepEnTitle);
```

Comment:
I'm having a hard reading this, maybe it would be nice to pass parameters on several line?

A suggestion (not sure how to deal with the pageTitle array):
(beware the indentation is wrong here)

```suggestion
      const result = await this.pageObjects.installPage.checkStepTitle(
        this.pageObjects.installPage.firstStepPageTitle,
        [
            this.pageObjects.installPage.firstStepFrTitle,
            this.pageObjects.installPage.firstStepEnTitle
        ]
    );
```